src/main.go

```diff
@@ -99,14 +102,22 @@ func main() {
 		endpointMetrics.StatusCode,
 		endpointMetrics.MonitorCount,
 	)
+	// go routine that reloads the entire map(cMap).
 	go func() {
-		for {
-			<-reload
-			log.Infoln("reloading...")
+		<-start
```

Comment:
sorry, i didnt get why is <-start here?

Reply:
The `start` channel simply triggers the goroutine on startup, indicating that the goroutine has started

Reply:
Sorry, but the `<-` is for receiving a signal, not sending a signal. It can block there.